docs/variants/protectli_vp2420/initial-deployment.md

```diff
@@ -0,0 +1,92 @@
+# Initial deployment
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom with
+the internal programmer. This document describes the process of building,
+installing and running flashrom on Ubuntu 22.04.
+
+## Deploy using Dasharo Tools Suite
+
+For simplicity we recommend using
+[Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite) to
+omit all compilation steps and deploy the Dasharo firmware seamlessly.
+Be sure to disable the BIOS lock in the AMI firmware setup utility:
+
+1. Go to `Chipset` tab
+2. Enter `PCH IO Configuration`
+3. Disable `BIOS Lock`.
+4. Save changes and reset.
+
+Now you are ready to use Dasharo Tools Suite (DTS):
+
+1. Boot Dasharo Tools Suite.
+2. Perform Dasharo installation.
+
+This will flash the full image, including the Intel ME. The operation requires
```

Comment:
I don't think we really need to flash full image. We didn't touch anything in ME region I believe for VP2420 (for VP2410 this procedure is required, but only for VP2410)). Simply disabling BIOS lock is enough. SO basically end the section with `Perform Dasharo installation`, the rest is obsolete

Reply:
Done.